src/plugin/timeline.js

```diff
@@ -376,7 +379,8 @@ export default class TimelinePlugin {
             this.params.secondaryLabelInterval
         );
 
-        let curPixel = 0;
+        let curPixel =
```

Comment:
you can simply use `this.params.offset` here (like line 376).

Reply:
True, thanks! 👍